spec/features/users/user_signs_up_spec.rb

```diff
@@ -43,6 +43,24 @@
     expect(page).to have_content(error_message('steamid.invalid'))
   end
 
+  scenario 'with out of range Steam ID' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.